resourceProcessor/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:17-jdk-alpine
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} app.jar
+ENTRYPOINT ["java","-jar","app"]
```

Comment:
It is a good practice to use "EXPOSE <PORT>" to describe what kind of ports are exposed from the Docker container.